libs/providers/flagd-web/README.md

```diff
@@ -0,0 +1,73 @@
+# flagd-web Provider for OpenFeature
+
+![Experimental](https://img.shields.io/badge/experimental-breaking%20changes%20allowed-yellow)
+
+A feature flag daemon with a Unix philosophy.
+
+## Installation
+
+:warning: This provider requires the use of the experimental @openfeature/web-sdk:
+
+```
+npm install @openfeature/web-sdk
+```
+
+```sh
+npm install @openfeature/flagd-web-provider
+```
+
+## Usage
+
+The `FlagdWebProvider` communicates with flagd via the [connect protocol](https://buf.build/blog/connect-a-better-grpc).
+
+### Available options
+
+| Option name | Type    | Default   | Description                                                                                                                                            |
+| ----------- | ------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| host        | string  | localhost | sets the host used to connect to the flagd instance                                                                                                    |
+| port        | number  | 8013      | sets the port used to connect to the flagd instance                                                                                                    |
+| tls         | boolean | false     | when set to true the provider will attempt to connect to flagd via https                                                                               |
+| maxRetries  | number  | 0         | sets the maximum number of retries for a connection to be made to the flagd instance, a value of 0 means unlimited. A negative value means no retries. |
```

Comment:
Sets the maximum number of retries for a connection to be made to the flagd instance 0 means unlimited. A negative number means no retries.